library/src/scala/quoted/Quotes.scala

```diff
@@ -246,7 +246,7 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
     type PackageClause <: Tree
 
     /** `TypeTest` that allows testing at runtime in a pattern match if a `Tree` is a `PackageClause` */
-    given PackageClauseTypeTest as TypeTest[Tree, PackageClause] = PackageClauseTypeTestImpl
+    given PackageClauseTypeTest: TypeTest[Tree, PackageClause] = PackageClauseTypeTestImpl
 
     /** Implementation of `PackageClauseTypeTest` */
     protected val PackageClauseTypeTestImpl: TypeTest[Tree, PackageClause]
```

Comment:
Looks like you could directly use abstract givens for this now:
```suggestion
    given PackageClauseTypeTest: TypeTest[Tree, PackageClause]
```

Reply:
@nicolasstucki ^

Reply:
Is that supported now?

Reply:
If it is supported I will create a PR to simplify this.